### ISSUE TITLE
`norm(packed_quantity, ...)` unpacks only to repack on non-Euclidean charts

### ISSUE THREAD

**Issue by nstarman** [GalacticDynamics/coordinax#690]
Found during the performance audit on #689. **Pre-existing, low priority, invisible under `jit`.**

## What

The metric-level overload for a packed `Quantity`:

```python
# _src/manifolds/norm.py
def norm(v: u.AbstractQuantity, metric, chart, /, *, at, usys=None):
    v_dict = cxcapi.cdict(v, chart)          # Quantity -> dict of Quantities
    return cxmapi.norm(v_dict, metric, chart, at=at, usys=usys)
```

and the CDict overload it delegates to immediately repacks:

```python
v_qm = cxcapi.carray(v, keys)                # dict of Quantities -> QuantityMatrix
```

So a caller who already has components packed in one `Quantity` pays an unpack and a repack to get back to a packed form.

## Measured

Eagerly, on `sph2` with `RoundMetric` (the Euclidean short-circuit does not apply):

```
norm generic cdict    sph2    5000 us
norm packed-Quantity  sph2    5625 us     <-- the packed form is SLOWER than the unpacked one
```

Backwards: passing data in the more structured form costs more than passing it looser.

## Scope

- **Invisible under `jit`** — this is Python-level packing overhead; the compiled graph is the same. A JAX library's hot paths are jitted, so this only affects interactive/eager use.
- **Euclidean + Cartesian is unaffected**: `norm(packed_quantity, cart3d)` short-circuits to `jnp.linalg.norm` and never reaches this path (confirmed by instrumenting the dispatch tree).
- Fix would be a direct `Quantity -> QuantityMatrix` route in the packed overload, skipping the dict round-trip.

Not urgent; filing so the measurement is not lost.

**Comment by nstarman** [comment]
Fixed in #691 — **1.49× eager, 1.11× trace**, `cdict` calls 2 → 0.

Correcting my own framing in the issue above: I filed this calling it *"low priority, invisible under `jit`"*. That was wrong on the second count. **Tracing executes the eager path**, so eager cost is compile cost — measured at 30–46% of trace+compile for these calls. The fix improves `jit` compilation, not just interactive use.

The 5625-vs-5000 µs figure in the issue was also a poor measurement (different call shapes, likely noise). The real cost is the round-trip itself: 1105 µs versus 27 µs to build the `QuantityMatrix` directly — 40×, and ~40% of the eager call.
